Tasks/DownloadBuildArtifacts/main.ts

```diff
@@ -70,7 +104,7 @@ async function main(): Promise<void> {
                 let downloader = new engine.ArtifactEngine();
                 var downloaderOptions = new engine.ArtifactEngineOptions();
                 downloaderOptions.itemPattern = itemPattern;
-                downloaderOptions.parallelProcessingLimit = +tl.getVariable("release.artifact.download.parallellimit") || 4;
+                downloaderOptions.parallelProcessingLimit = +tl.getVariable("release.artifact.download.parallellimit") || 8;
```

Comment:
no need to set default if not there as ILD will set it

Reply:
done.